SR2020/2019-11-25-kit-shipment-update.md

```diff
@@ -0,0 +1,12 @@
+---
+to: Those who are awaiting
+subject: Kit Shipment update
+---
+
+Hello!
+
+Unfortunately, We are currently facing delays getting your kit shipped out to you.
+
+Please be assured we're actively working on this, and will let you know as there's an update. We have also taken steps to ensure this delay doesn't happen in future.
```

Comment:
nit: "have" implies those steps are complete, perhaps "are" or "will be"?

Reply:
I was more talking about the fact we'd know about these issues sooner, now the kit / competition team split is _slightly_ better defined.

But I agree the wording here can be improved !